mpd/mpd.go

```diff
@@ -218,19 +263,31 @@ type AudioChannelConfiguration struct {
 
 // Creates a new MPD object.
 // profile - DASH Profile (Live or OnDemand).
-// mediaPresentationDuration - Media Presentation Duration (i.e. PT6M16S).
 // minBufferTime - Min Buffer Time (i.e. PT1.97S).
-func NewMPD(profile DashProfile, mediaPresentationDuration string, minBufferTime string) *MPD {
+// attributes - Other attributes (optional)
+func NewMPD(profile DashProfile, minBufferTime string, attributes ...AttrMPD) *MPD {
```

Comment:
Note this makes this a breaking change for any downstream usages

Reply:
Yes, I know. Should I use another method name?

Reply:
Also from ISO23009 mediaPresentationDuration is a mandatory field for type "static" which is the default here.

Reply:
Exactly, you are correct. I found this method creates only static type.

Is it a solution that uses two methods?

Example
```go
func NewMPD(profile DashProfile, mediaPresentationDuration, minBufferTime string, attributes ...AttrMPD) *MPD
```

```go
func NewMPDDynamic(profile DashProfile, minBufferTime string, attributes ...AttrMPD) *MPD
```

Reply:
or `NewDynamicMPD` ?